### DISCUSSION TITLE
SESAC Blocks Music Videos on YouTube. Does this affect FreeTube as well?

### DISCUSSION THREAD

**Discussion by liammcsh2008** [FreeTubeApp/FreeTube#5765] [General]
So I recently found out many you sick videos on YouTube, were deleted due to a dispute between them, and SEASAC. Is free tube‘s Local API affected. Because the majority of videos contain music from them.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
FreeTube is showing everything from YT as you were browsing the site in a logged out state. So if you dont see the content on YT, you wont see it in FreeTube
